### DISCUSSION TITLE
Running nx with Yarn 2

### DISCUSSION THREAD

**Discussion by jvskriubakken** [nrwl/nx#7243] [Show and Tell]
Hi!

I am wondering, can I run nx with yarn 2?

**Comment by BeyondEvil** [comment]
I have the same question (but with Yarn 3).

**Comment by EfstathiadisD** [comment]
There is a PR implementing support but it was huge. The Nx team will probably split it up into multiple ones and it gonna take a bit. Yarn 2/3 is a huge departure. Probably with the Nodelinker it's gonna work without problems but Zero install is a bit away..

**Reply by BeyondEvil** [reply]
Yeah, I got it to work with Nodelinker, but it would be really nice to have ZI support.

**Comment by MrChrisRodriguez** [comment]
We use Yarn 3 in the `node_modules` configuration with hoisting and it's a huge improvement over Yarn Classic. But yeah, iirc ZI isn't working.
